lstchain/reco/utils.py

```diff
@@ -649,12 +655,37 @@ def get_effective_time(events):
     t_eff: float
     t_elapsed: float
     """
-    
+
+    # time differences between the events in the table (which in general are
+    # NOT all triggered events):
+    time_diff = np.diff(events.dragon_time)
+    # elapsed time: sum of those time differences, excluding large ones which
+    # might indicate the DAQ was stopped (e.g. if the table contains more
+    # than one run). We set 0.1 s as limit to decide a "break" occurred:
+    t_elapsed = np.sum(time_diff[time_diff<0.1])
+
+    # Get delta_t, the time elapsed since the previous triggered event.
+    # We exclude the first event, for which delta_t might be set to 0 if it is
+    # the first even in a file.
+    # We want this to calculate the actual trigger rate, on which the dead
+    # time depends.
     delta_t = events.delta_t[1:]
```

Comment:
As mentioned above, it should be `events["delta_t"]`

Reply:
> As mentioned above, it should be `events["delta_t"]`

ok!